index.js

```diff
@@ -65,6 +65,14 @@ function formatWithCursor(text, opts, addAlignmentSize) {
   if (opts.requirePragma && !hasPragma(text)) {
     return { formatted: text };
   }
+  if (
+    opts.prependPragma &&
+    !hasPragma(text) &&
+    opts.rangeStart === 0 &&
+    opts.rangeEnd === Infinity
+  ) {
+    text = `/** @format */\n${text}`;
```

Comment:
If there's a docblock like `/** @flow */` you want to merge into it, rather than prepending above it.

Current in:
```js
/** @flow */
```

Current out:
```js
/** @format */
/** @flow */
```

Expected out:

```js
/**
 * @flow
 * @format
 */
```

Reply:
Ideally there'd be a `require('jest-docblock').print(obj)`

Reply:
I actually went ahead and implemented this here: https://github.com/facebook/jest/pull/4517

Reply:
Should we consider another name (e.g. `insertPragma`) for this option? since it **prepend** pragma if there's no top docblock, otherwise **append** pragma to the top docblock.

Reply:
I like insert better :)